### PR TITLE
Renamed data provider to supplier everywhere

### PR DESCRIPTION
**Pull request by mingazhev**

### CODE REVIEW

**Review by kirillta — APPROVED** [review]

File: HappyTravel.LocationUpdater/Services/LocationUpdaterHostedService.cs [hunk 1]
```diff
@@ -115,14 +115,14 @@ private async Task DownloadAndMergeLocations()
...
+            foreach (var (supplierName, supplierValue) in GetSuppliers(_options.Suppliers))
             {
-                await DownloadAndAddLocationsToDb(providerName, providerValue, lastModified);
+                await DownloadAndAddLocationsToDb(supplierName, supplierValue, lastModified);
             }
         }
 
 
-        private async Task DownloadAndAddLocationsToDb(string providerName, DataProviders providerType,
+        private async Task DownloadAndAddLocationsToDb(string providerName, Suppliers providerType,
```
Comment (hasmikharutyunyan) [inline HappyTravel.LocationUpdater/Services/LocationUpdaterHostedService.cs:125]: maybe providerName?

File: HappyTravel.LocationUpdater/Services/LocationUpdaterHostedService.cs [hunk 2]
```diff
@@ -149,7 +149,7 @@ private async Task DownloadAndMergeLocations()
         }
 
 
-        private async Task UploadLocationsToDb(DataProviders providerType, List<Location> locations)
+        private async Task UploadLocationsToDb(Suppliers providerType, List<Location> locations)
```
Comment (hasmikharutyunyan) [inline HappyTravel.LocationUpdater/Services/LocationUpdaterHostedService.cs:152]: providerType

Comment (mingazhev) [reply HappyTravel.LocationUpdater/Services/LocationUpdaterHostedService.cs:152]: fixed

File: HappyTravel.LocationUpdater/Services/LocationUpdaterHostedService.cs (hunk 1 above)
Comment (mingazhev) [reply HappyTravel.LocationUpdater/Services/LocationUpdaterHostedService.cs]: fixed

**Review by hasmikharutyunyan — APPROVED** [review]

**Pull request merged**